tests/unit/server/api/v1/test_records.py

```diff
@@ -71,932 +70,1002 @@ async def create_ranking_question(dataset: "Dataset") -> None:
     await RankingQuestionFactory.create(name="ranking_question_2", dataset=dataset)
 
 
-@pytest.mark.parametrize("response_status", ["submitted", "discarded", "draft"])
-@pytest.mark.parametrize(
-    "create_questions_func, responses",
-    [
-        (
-            create_text_questions,
-            {
-                "values": {
-                    "input_ok": {"value": "yes"},
-                    "output_ok": {"value": "yes"},
-                },
-            },
-        ),
-        (
-            create_rating_questions,
-            {
-                "values": {
-                    "rating_question_1": {"value": 5},
+@pytest.mark.asyncio
+class TestSuiteRecords:
```

Comment:
Great work! I think that following this `TestSuite...` approach is the way to go 💪🏻